target/linux/ramips/dts/mt7621_zte_mf286c3.dts

```diff
@@ -0,0 +1,333 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "zte,mf286c3", "mediatek,mt7621-soc";
+	model = "ZTE MF286C3";
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+		bootargs-override = "console=ttyS0,57600n8";
+	};
+
+	aliases {
+		led-boot = &led_status;
+		led-failsafe = &led_status;
+		led-running = &led_status;
+		led-upgrade = &led_status;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		pinctrl-names = "default";
+		pinctrl-0 = <&led_pins>;
+
+		led-6  { // p0_0 sig00
+			label = "blue:sig_0";
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&aw9523 0 GPIO_ACTIVE_LOW>;
+		};
+
+		led-7 { // p0_1 sig01
+			label = "blue:sig_1";
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&aw9523 1 GPIO_ACTIVE_LOW>;
+		};
+
+		led-8 { // p0_2 sig02
+			label = "blue:sig_2";
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&aw9523 2 GPIO_ACTIVE_LOW>;
+		};
+
+		led-9 { // p0_3 - sig03
+			label = "blue:sig_3";
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&aw9523 3 GPIO_ACTIVE_LOW>;
+		};
+
+		led-3 { // p0_5 - rgb05-red
+			label = "rgb-red";
+			color = <LED_COLOR_ID_RED>;
+			gpios = <&aw9523 5 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status: led-0 { // p0_6 - power_06
+			label = "blue:power";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_STATUS;
+			gpios = <&aw9523 6 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		led-4 { // p1_0 - rgb10-blue
+			label = "rgb-blue";
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&aw9523 8 GPIO_ACTIVE_LOW>;
+		};
+
+		led-10 { // p1_1 - sig11
+			label = "blue:sig_5";
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&aw9523 9 GPIO_ACTIVE_LOW>;
+		};
+
+		led-1 { // p1_2 - wifi12
+			label = "blue:wifi";
+			color = <LED_COLOR_ID_BLUE>;
+			function = LED_FUNCTION_WLAN;
+			gpios = <&aw9523 10 GPIO_ACTIVE_LOW>;
+		};
+
+		led-5 { // p1_3 - rgb13-green
+			label = "rgb-green";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&aw9523 11 GPIO_ACTIVE_LOW>;
+		};
+
+		led-2 { // p1_4 - phone14
+			label = "blue:phone";
+			color = <LED_COLOR_ID_BLUE>;
+			gpios = <&aw9523 12 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <50>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&button_pins>;
+
+		p1_6 {
```

Comment:
Please follow the pattern  "^(button|event|key|switch|(button|event|key|switch)-[a-z0-9-]+|[a-z0-9-]+-(button|event|key|switch))$".
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/input/gpio-keys.yaml